err.go

```diff
@@ -8,6 +8,6 @@ type PermissionError struct {
 	what string
 }
 
-func (self PermissionError) Error() string {
-	return fmt.Sprint("Unable to access record:", self.what)
+func (pe PermissionError) Error() string {
+	return fmt.Sprint("Unable to access record:", pe.what)
```

Comment:
Instead of Sprint, you can just `return "Unable to access record: " + pe.what`

Reply:
Github Copilot 😅  will change